src/neigh.c

```diff
@@ -1101,7 +1101,20 @@ static struct dpvs_sockopts neigh_sockopts = {
 };
 
 #define NEIGH_LCORE_JOB_MAX     2
-static struct dpvs_lcore_job neigh_jobs[NEIGH_LCORE_JOB_MAX];
+static struct dpvs_lcore_job neigh_jobs[NEIGH_LCORE_JOB_MAX] = {
+    [0] = {
+        .name = "neigh_sync",
+        .type = LCORE_JOB_SLOW,
+        .func = neigh_process_ring,
+        .skip_loops = NEIGH_PROCESS_MAC_RING_INTERVAL,
+    },
+
+    [1] = {
+        .name = "neigh_sync",
+        .type = LCORE_JOB_LOOP,
+        .func = neigh_process_ring,
+    }
+};
```

Comment:
It's a little confusing because a job seems to be registered twice. You could make use of `dpvs_lcore_role_t` to fix it, just as what does in netif.

Reply:
Done as requested.